_episodes/16-file-formats.md

```diff
@@ -46,6 +46,15 @@ parameterization.
 ~~~
 {: .source}
 
+___Note:___ To follow the example below, you need to download the example input file, *file-formats.bam*. The file is available from [___Note:___ To follow the example below, you need to download the example input file, *file-formats.bam*. The file is available from [https://github.com/common-workflow-language/user_guide/blob/gh-pages/_includes/cwl/file-formats.bam
+](https://github.com/common-workflow-language/user_guide/blob/gh-pages/_includes/cwl/file-formats.bam) and can be downloaded e.g. via `wget`:
+
+~~~
+wget https://github.com/common-workflow-language/user_guide/blob/gh-pages/_includes/cwl/file-formats.bam
```

Comment:
wget on https://github.com/common-workflow-language/user_guide/raw/gh-pages/_includes/cwl/file-formats.bam (otherwise you get a webpage)

Reply:
No worries, thank you for your continuing contributions!